### ISSUE TITLE
Remove all uses of "More" and "Again" by the QM

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#28]
TLDR: The "More" and "Again" prompts are used inconsistently by quizmasters, which results in unfairness, and because of the inconsistent usage, really results in little useful information being provided to the quizzer. 

If a quizzer is answering, no ruling has happened, and time has not elapsed, they know they need to keep providing information. That's everything the "more" prompt provides. "Again" is only used on F/Q questions, and can only be used at the end of the verse anyway. 

Removing their usages altogether has these positive effects:
* Far less work for a QM to do
* Perfect consistency across rooms (since there's no more "more" or "again" prompts!!)

Cons:
* For lesser experienced quizzers, these prompts serve as a gentle nudge to "keep answering." I would argue that this is merely a coaching point. If everyone knew QMs could not provide these prompts anymore, it would be a coaching emphasis. And the coaching emphasis is a very positive one. It empowers the quizzer. "Keep answering if you have time left and no ruling has happened." It's simple and applies to all situations.

**Comment by jttower** [comment]
Scott, as a point of clarification, on Q/F questions, if we remove the "again" prompt, then what do you suggest would replace it?  Or are you saying that the quizzer should just know to start their verse over if the QM doesn't say anything?  I think the "again" prompt should only be used in Q/F question types.  I don't think we should remove it altogether.  

I agree about the "more" prompt.  I think it is confusing and I agree it is used more as a coaching point, especially for new quizzers.  I think using "more" as the prompt for "I need more of the question" is confusing.  Though quizzers can be trained, it just feels weird every time I use that prompt in looking for them to finish saying "14" because I only said "fourte-".  I would rather it be changed to "what is your question" like we do for Reference questions when the question has not been completely read.  This feels more natural and would simplify the number of prompts.

**Comment by ARMediting** [comment]
I'm thoroughly against changing a rule simply because quizmasters apply it inconsistently. How to use prompts is clearly spelled out in the RB. 

However, I agree that the current uses of "More" are very confusing to many quizzers. There needs to be some short way for quizzers to prompt for the finishing of a verse number, as Jessica referenced, but otherwise the usages seem arbitrary. Quizzers get messed up as least as often when the _answer_ comes before the question, and there's no prompt for that.

Could we keep "Again" for when a quizzer has substantially given everything needed in question and answer but needs to fix something inside what they've said; and drop "More" altogether? This could even be a better nudge for quizzers than improper uses of "More," because it gives them a very clear sense of what needs to happen: more quoting or a correction. (Of course both could be needed, but that already happens and is already messy.)

**Comment by kclimenhaga** [comment]
I agree that removing "more" as a prompt would make the prompts less arbitrary. I am in favour of "What is your reference?" on a quote or CVR when it is needed.

**Comment by jswingle** [comment]
I agree with the removal of "more" and "again," with the exception of "again" in F/Q questions. Mainly because (1) this does actually provide useful information to the quizzer, (2) it's already being applied objectively and consistently.

I think the full list of prompts should be:

- "Again" - when the quizzer reaches the end of the designated verse(s) in a F/Q but needs to correct their answer
- "Question?" - when the quizzer needs to give their question on a CR/CVR (reference issue #133 )
- "Clarify PRONOUN" - when the quizzer needs to clarify a pronoun
- "Quote is complete" - when the quizzer finishes quoting the quotation of a SIT

I'm open to @kclimenhaga 's idea of a "what is your reference?" (or just "reference?").

**Comment by scottpeterson** [comment]
> Scott, as a point of clarification, on Q/F questions, if we remove the "again" prompt, then what do you suggest would replace it? 

Nothing. The prompt adds nothing. It's trivial to coach quizzers "if the quizmaster has not made a ruling, and if your 30 seconds are still going, keep answering."

**Comment by scottpeterson** [comment]
> I agree with the removal of "more" and "again," with the exception of "again" in F/Q questions. Mainly because (1) this does actually provide useful information to the quizzer, (2) it's already being applied objectively and consistently.

Is the quizmaster required to say "again" if the quizzers repeats the last 3 words of the verse over and over again?  Technically they've "reached the end of the verse."

**Comment by scottpeterson** [comment]
If a quizzer has said everything word perfect, but omitted a phrase _in the middle_,  quizmasters are very mixed on whether they use "again" or "more."

**Comment by scottpeterson** [comment]
> Could we keep "Again" for when a quizzer has substantially given everything needed in question and answer but needs to fix something inside what they've said; and drop "More" altogether? This could even be a better nudge for quizzers than improper uses of "More," because it gives them a very clear sense of what needs to happen: more quoting or a correction. (Of course both could be needed, but that already happens and is already messy.)


If we drop "more" and "again" can only be used in this one scenario, then what information is it providing to quizzers?

**Comment by jttower** [comment]
I like Jeremy's list of 4 prompts, and I think it can include Kristin's issue.  If the question/reference has not been finished (like my example of "fourt..."), that means something in the question is missing, so QM prompts "Question?".  

However, I can also see an argument of including "Reference?" simply because I could potentially see "Question?" being misused if some other information is missing from the question.  I believe currently in this situation QM's should stay silent, since not all the information in the question and answer has been given by the quizzer.  For this reason, I would agree to add "Reference?" to the list.

**Comment by jttower** [comment]
See Issue #175 for an update on this issue.

**Comment by jttower** [comment]
As was posted on Issue #175 , at the Rules Committee meeting on 4/2/23, we voted 5-1 to change to the following:
—-----------------------------------
3.2.1.1. Quizmaster Prompts
During the answering period, the quizmaster will prompt the quizzer in the following ways. If the quizmaster has given a prompt that is clear and is in keeping with the spirit of these rules even if not word for word, it is acceptable as long as the occasion for the prompt is still correct.
3.2.1.1.1. Clarify
If a quizzer has provided all the information in the question and answer, except the identification of a pronoun present which is part of the answer text, the quizmaster will prompt the quizzer to identify the specific pronoun. For example: "Can you clarify 'He'?" or "Clarify 'They.'"
The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification. A pronoun may need to be identified only if the antecedent is in context.
3.2.1.1.2. What is your Reference?
On a Quote or Reference type question, if the quizmaster did not finish saying the entire reference before the quizzer jumped, then the quizmaster will prompt for the reference. For example: "What is your reference?" or "Your reference?".
3.2.1.1.3. What is your Question?
On a Reference type question, if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will prompt for the question. For example: "What is your question?" or "Question?".
3.2.1.1.4. Again
On a Quote or Finish type question, once the quizzer has quoted to the end of the required verse or verses, if the answer is not yet word-perfect, the quizmaster will say, "Again." This will occur as often as needed until the quizzer's answer is word-perfect or the answering period expires. The quizmaster is not allowed to say "Again" until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
3.2.1.1.5. Quote is Complete
On a Situation type question, when the quizzer has completed the quotation, the quizmaster will tell the quizzer that the quote is finished. For example: "Quote is complete” or “That is the quote.”
